### ISSUE TITLE
Microsoft - error saying that the account is already used on a newly created alias

### ISSUE THREAD

**Issue by sbadau** [mozilla/fx-private-relay#444]
**Affected versions**
Firefox 77.0.1
Firefox Private Relay 1.3.1

**Affected platforms**
Ubuntu 18.04
Mac OS X 10.5
Windows 10

**Steps to reproduce**:
1. Launch Firefox.
2. Install the Firefox Private Relay add-on
3. Create a new relay alias and copy it.
4. Navigate to Microsoft.com, reach the sign-up form
5. Create an account using the copied alias.

**Expected behavior**
A new account should be created with the copied alias.

**Actual behavior**
I get an error that says that for that alias there already is a Microsoft account and to try using a different address. 
Please see the screencast for more details.

![Microsoft (1)](https://user-images.githubusercontent.com/56026823/85003745-4cfed500-b15f-11ea-90da-e3b224b8fb8b.gif)

**Comment by groovecoder** [comment]
It looks like Microsoft is blocking email addresses with many subdomains. I don't think this happens in production - i.e., relay.firefox.com is a "small enough" domain that Microsoft allows it. But we'll check this in prod after we launch or latest code there.

**Comment by groovecoder** [comment]
Checked on relay.firefox.com and the addresses work. Closing this as it only affects dev & stage servers.

**Comment by sbadau** [comment]
I can confirm that this issue is no longer reproducible with the latest version of the Private Relay add-on (1.4.1) and Firefox 77.0.1 (verified on Mac OS X 10.15, Ubuntu 18.04 and Windows 10). However, even though the error saying that the account is already used when using a relay alias, is not seen anymore, I did not receive an email confirmation to finalize the account creation. I logged issue #511 for that matter.
